File: Stack_Dynamic.cpp
#include<iostream>
using namespace std;

void testStackDynamic(int n) {
    int arr[n]; // VLA (Variable Length Array) on stack
    for(int i = 0; i < n; i++) 
        arr[i] = i * 2;
    
    for(int i = 0; i < n; i++) 
        cout << arr[i] << " ";
}

int main() {
    int n;
    cout << "Enter size: ";
    cin >> n;
    testStackDynamic(n);
    return 0;
}

File: Stack_Dynamic.js
function testStackDynamic(n) {
    let arr = new Array(n);
    for(let i = 0; i < n; i++) 
        arr[i] = i * 2;
    
    console.log(arr);
}

let n = parseInt(prompt("Enter size:"));
testStackDynamic(n);
